From d5e08f63b1ca6746a51e62cda4df5483c0d3e2ec Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@pbarker.dev>
Date: Wed, 21 Oct 2026 07:16:23 +0100
Subject: [PATCH] Blacklist "emSign ECC Root CA - G3" and "Comodo AAA Services
 root"

[PB: Backported to address the eMudhra Intermediate CA Compromise
     disclosed at https://emudhra.we-have-a-problem.com. As this is a
     certificate authority compromise rather than a software flaw it
     doesn't have a CVE number, but this should be treated like a CVE
     fix.]

Upstream-Status: Backport [https://salsa.debian.org/debian/ca-certificates/-/commit/4a0203e5ce4ce1a064b97387f52ca6a00c97185c]
Signed-off-by: Paul Barker <paul@pbarker.dev>
---
 mozilla/blacklist.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/mozilla/blacklist.txt b/mozilla/blacklist.txt
index d84ceb919906..ba3635f7af87 100644
--- a/mozilla/blacklist.txt
+++ b/mozilla/blacklist.txt
@@ -3,4 +3,9 @@
 # Blacklist explicitly distrusted certificates to explicitly ignore them and prevent build errors
 "Explicitly Distrust DigiNotar Root CA"
 
+# Blacklist "emSign ECC Root CA - G3" (cf. #1151413)
+"emSign ECC Root CA - G3"
+# Blacklist "Comodo AAA Services root" (cf. #1151413)
+"Comodo AAA Services root"
+
 # Expired CAs
-- 
2.43.0

